src/vt/configs/arguments/args.cc

```diff
@@ -185,6 +193,241 @@ std::tuple<int, std::string> parseArguments(
+  appConfig.vt_lb_statistics_compress = lb_stats["Enable Compression"].as<bool>(false);
+  appConfig.vt_lb_statistics_file = lb_stats["File"].as<std::string>("vt_lb_statistics.%t.json");
+  appConfig.vt_lb_statistics_dir = lb_stats["Directory"].as<std::string>("");
+
+  appConfig.vt_lb_self_migration = load_balancing["Enable Self Migration"].as<bool>(false);
+  appConfig.vt_lb_spec = load_balancing["Enable Specification"].as<bool>(false);
+  appConfig.vt_lb_spec_file = load_balancing["Specification File"].as<std::string>("");
+  // appConfig.vt_lb_run_lb_first_phase = load_balancing["vt_lb_run_lb_first_phase"].as<bool>(false);
+
+  // Diagnostics
+  YAML::Node diagnostics = yaml_input["Diagnostics"];
+  bool default_diag_enable;
+#if (vt_diagnostics_runtime != 0)
+  default_diag_enable = true;
+#else
+  default_diag_enable = false;
+#endif
+  appConfig.vt_diag_enable = diagnostics["Enabled"].as<bool>(default_diag_enable);
+  appConfig.vt_diag_print_summary = diagnostics["Enable Print Summary"].as<bool>(false);
+  appConfig.vt_diag_summary_file = diagnostics["Summary File"].as<std::string>("vtdiag.txt");
+  appConfig.vt_diag_summary_csv_file = diagnostics["Summary CSV File"].as<std::string>("");
+  appConfig.vt_diag_csv_base_units = diagnostics["Use CSV Base Units"].as<bool>(false);
+
+  // Termination
+  YAML::Node termination = yaml_input["Termination"];
+  appConfig.vt_no_detect_hang = not termination["Detect Hangs"].as<bool>(true);
+  appConfig.vt_term_rooted_use_ds = termination["Use DS for Rooted"].as<bool>(false);
+  appConfig.vt_term_rooted_use_wave = termination["Use Wave for Rooted"].as<bool>(false);
+  appConfig.vt_epoch_graph_on_hang = termination["Output Epoch Graph on Hang"].as<bool>(true);
+  appConfig.vt_epoch_graph_terse = termination["Terse Epoch Graph Output"].as<bool>(false);
+  appConfig.vt_print_no_progress = termination["Print No Progress"].as<bool>(true);
+  appConfig.vt_hang_freq = termination["Hang Check Frequency"].as<int64_t>(1024);
+
+  // Debugging/Launch
+  YAML::Node launch = yaml_input["Launch"];
+  appConfig.vt_pause = launch["Pause"].as<bool>(false);
+
+  // User Options
+  YAML::Node user_options = yaml_input["User Options"];
+  int bool_iter = 1, int_iter = 1, str_iter = 1;
+  bool too_many_user_args = false;
+  for (YAML::const_iterator it = user_options.begin(); it != user_options.end(); ++it) {
+    const std::string& key = it->first.as<std::string>();
+    try {
+      bool user_input = user_options[key].as<bool>();
+      if (bool_iter == 1) appConfig.vt_user_1 = user_input;
+      else if (bool_iter == 2) appConfig.vt_user_2 = user_input;
+      else if (bool_iter == 3) appConfig.vt_user_3 = user_input;
+      else too_many_user_args = true;
+      appConfig.user_args["vt_user_" + std::to_string(bool_iter)] = key;
+      bool_iter++;
+    } catch(...) {
+      try {
+        int user_input = user_options[key].as<int>();
+        if (int_iter == 1) appConfig.vt_user_int_1 = user_input;
```

Comment:
Place the instruction in a separate line after an `if` or `else`:
```suggestion
        if (int_iter == 1)
          appConfig.vt_user_int_1 = user_input;
```

Reply:
When in doubt you can use `clang-format` (for example `git clang-format HEAD~1`). Note that the `.clang-format` configuration file we have does not reflect `vt`'s formatting style perfectly :shrug:

Reply:
Fixed, thanks